app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -56,7 +57,11 @@ fun HudLocation(location: String) {
     ) {
         Text(
             text = location,
-            modifier = Modifier.align(Alignment.TopCenter)
+            modifier = Modifier
+                .align(Alignment.TopCenter).testTag("locationText")
```

Comment:
We've agreed on writing test tags with snake case rather than camel case.

Reply:
My bad, addressed!